viewer/decryptPcap.js

```diff
@@ -88,6 +88,7 @@ async function premain () {
   Db.initialize({
     host: escInfo,
     prefix: Config.get('prefix', 'arkime_'),
+    queryExtraIndices: Config.getArray('queryExtraIndices', ''),
```

Comment:
`Db.initialize` now needs to be passed the `queryExtraIndices` array value from config, defaulting to empty if not found.